README.md

```diff
@@ -104,81 +112,7 @@ c12 merged config sources with [unjs/defu](https://github.com/unjs/defu) by belo
 
 ## Options
 
-### `cwd`
```

Comment:
This needs to be kept.

Reply:
We have [automd](https://github.com/unjs/automd) generator for jsdocs. (see [ufo](https://github.com/unjs/ufo) for example is using it)

For c12, I am happy with current manual readme to have additional docs. We might in the future use jsdocs to auto maintain readme but not for now.